src/games/strategy/engine/framework/mapDownload/InstallMapDialog.java

```diff
@@ -302,11 +302,12 @@ private void install(final DownloadFileDescription selected, final int count, fi
       Util.notifyError(this, download.getError());
       return;
     }
-
-    try ( FileOutputStream sink = new FileOutputStream(tempFile)){
+    try {
       validateZip(download);
```

Comment:
Move this validateZip to be before we open a file output stream. Otherwise we still validate before calling `sink.write`, so prior behavior maintained there..